dvc/stage.py

```diff
@@ -522,13 +522,15 @@ def load(repo, fname):
         return stage
 
     def dumpd(self):
+        from dvc.remote.local import RemoteLOCAL
+
         return {
             key: value
             for key, value in {
                 Stage.PARAM_MD5: self.md5,
                 Stage.PARAM_CMD: self.cmd,
-                Stage.PARAM_WDIR: os.path.relpath(
-                    self.wdir, os.path.dirname(self.path)
+                Stage.PARAM_WDIR: RemoteLOCAL.unixpath(
```

Comment:
let's move `unixpath` out of RemoteLOCAL, into a `utils/path` or something else in utils. It looks weird that `RemoteLOCAL` is being used as a library.

Reply:
@shcheklein , I'm thinking of using `Path` instead of `os.path`, is it OK if I submit a patch later to reduce all this _path logic_?

Reply:
@mroutis sure!